README.md

```diff
@@ -110,5 +106,10 @@ with Accumulo 1.6.2 and 1.7.0 all API changes will follow [semver 2.0][12]
 [7]: http://thrift.apache.org/
 [8]: http://accumulo.apache.org/notable_features.html
 [9]: http://maven.apache.org/
+[10]: http://accumulo.apache.org/user_manual.html
```

Comment:
These should use https wherever possible. I know some of the older links don't have it, but they should. At the very least, new ones should.